core/templates/dev/head/services/PlaythroughIssuesService.js

```diff
@@ -82,23 +82,31 @@ oppia.factory('PlaythroughIssuesService', [
     return {
       initSession: function(
           newExplorationId, newExplorationVersion,
-          newWhitelistedExplorationIds) {
+          testOnlyWhitelistedExplorationIds) {
         explorationId = newExplorationId;
         explorationVersion = newExplorationVersion;
-        whitelistedExplorationIds = newWhitelistedExplorationIds || [];
+        if (testOnlyWhitelistedExplorationIds) {
+          whitelistedExplorationIds = testOnlyWhitelistedExplorationIds;
+        } else {
+          PlaythroughIssuesBackendApiService
+            .fetchWhitelistedExplorationsForPlaythroughs().then(
+              function(newWhitelistedExplorationIds) {
+                whitelistedExplorationIds = newWhitelistedExplorationIds;
+              });
+        }
       },
-      isExplorationWhitelisted: function(explorationId) {
+      isExplorationEligibleForPlaythroughIssues: function(explorationId) {
         return whitelistedExplorationIds !== null &&
```

Comment:
What happens if the whitelist is empty? 

What is the expected final state of the whitelist, when we launch this feature? Maybe it will be useful to have a flag somewhere to enable/disable the whitelist? (This will provide us with more control while releasing, as all the coordinator has to do is flip a flag to disable whitelisting, and enable playthroughs for all exps).

Reply:
If the whitelist is empty, then no explorations are eligible.

When we launch this feature, all usages of this whitelist will be removed from the code base. I don't think a control for it will be necessary since it's strictly temporary?

Reply:
Ah I think here a flag would be unnecessary. Thanks for pointing it out. So just confirming from the release pov. The release when we want to launch this feature, we just delete any usage of the whitelist, and the feature should be accessible to all right?

Reply:
Yup, that's right!